kratos/python/add_strategies_to_python.cpp

```diff
@@ -519,6 +548,35 @@ namespace Kratos
                     .def("Info", &BaseSolvingStrategyType::Info)
                     ;
 
+            typedef ExplicitSolvingStrategy< SparseSpaceType, LocalSpaceType > BaseExplicitSolvingStrategyType;
+            py::class_<BaseExplicitSolvingStrategyType, typename BaseExplicitSolvingStrategyType::Pointer>(m, "ExplicitSolvingStrategy")
+                .def(py::init<ModelPart &, bool, int>())
+                .def(py::init<ModelPart&, typename ExplicitBuilderType::Pointer, bool, int>())
+                .def("Predict", &BaseExplicitSolvingStrategyType::Predict)
+                .def("Initialize", &BaseExplicitSolvingStrategyType::Initialize)
+                .def("IsConverged", &BaseExplicitSolvingStrategyType::IsConverged)
+                .def("CalculateOutputData", &BaseExplicitSolvingStrategyType::CalculateOutputData)
+                .def("SetEchoLevel", &BaseExplicitSolvingStrategyType::SetEchoLevel)
+                .def("GetEchoLevel", &BaseExplicitSolvingStrategyType::GetEchoLevel)
+                .def("SetRebuildLevel", &BaseExplicitSolvingStrategyType::SetRebuildLevel)
+                .def("GetRebuildLevel", &BaseExplicitSolvingStrategyType::GetRebuildLevel)
+                .def("SetMoveMeshFlag", &BaseExplicitSolvingStrategyType::SetMoveMeshFlag)
+                .def("MoveMeshFlag", &BaseExplicitSolvingStrategyType::MoveMeshFlag)
+                .def("MoveMesh", &BaseExplicitSolvingStrategyType::MoveMesh)
+                .def("Clear", &BaseExplicitSolvingStrategyType::Clear)
+                .def("Check", &BaseExplicitSolvingStrategyType::Check)
+                .def("InitializeSolutionStep", &BaseExplicitSolvingStrategyType::InitializeSolutionStep)
+                .def("FinalizeSolutionStep", &BaseExplicitSolvingStrategyType::FinalizeSolutionStep)
+                .def("SolveSolutionStep", &BaseExplicitSolvingStrategyType::SolveSolutionStep)
+                //.def("GetModelPart", &BaseExplicitSolvingStrategyType::GetModelPart )
```

Comment:
this is necessary imo, why commented?

Reply:
It is commented in the base implicit one? (possibly to fix the compilation because of the `const`s). I will activate it in here and we can amend the base one once we unify the strategies.